src/database/migrations/1739918380436_member_user_public_key.ts

```diff
@@ -0,0 +1,19 @@
+import { type Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+    await db.schema
+        .createTable('member_user_public_key')
+        .addColumn('id', 'uuid', (col) => col.defaultTo(sql`v7uuid()`).primaryKey())
```

Comment:
IDK if we need this (e.g. if we plan to return an ID value to users via API), but included it as the PK for now.